### PR TITLE
feat(spamd): migrate code

### PR DESCRIPTION
**Pull request by PierrickV**
No changelog

### CODE REVIEW

File: SpamAssassin/spamassassin/ingest/parser.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            prefork: %{GREEDYDATA}
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          os.family: "linux"
+          os.platform: "linux"
```
Comment (squioc) [inline SpamAssassin/spamassassin/ingest/parser.yml]: I think we can remove this part.
```suggestion
```

File: SpamAssassin/spamassassin/ingest/parser.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          source.domain: "{{event.message.source_domain}}"
+          source.ip: "{{event.message.source_ip}}"
+          source.port: "{{event.message.source_port}}"
+          source.size_in_char: "{{event.message.source_domain | length}}"
```
Comment (squioc) [inline SpamAssassin/spamassassin/ingest/parser.yml:36]: I suggest removing it
```suggestion
```

File: SpamAssassin/spamassassin/_meta/fields.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+source.size_in_char:
+  description: ""
+  name: source.size_in_char
+  type: number
```
Comment (squioc) [inline SpamAssassin/spamassassin/_meta/fields.yml:4]:
```suggestion
```

**Review by squioc — APPROVED** [review]
Thanks for the job

**Comment by PierrickV** [comment]
@squioc thank you for your review :+1: 

I applied your suggestions =D

**Review by squioc — APPROVED** [review]

**Pull request merged**